recipes/jupyterlab_code_formatter/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "jupyterlab_code_formatter" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/ryantam626/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 923efce4fb9bae3196218624bb7e5e3f0f064b73eda01ef2cd60b9ad307beeaf
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  build:
+    - nodejs
+  host:
+    - jupyterlab
+    - pip
+    - python
+  run:
+    - notebook
+    - python >=3.6
+
+test:
+  imports:
+    - jupyterlab_code_formatter
+
+about:
+  home: https://github.com/ryantam626/jupyterlab_code_formatter
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: The server extension for jupyterlab_code_formatter.
```

Comment:
Could be changed to `summary: A universal code formatter for JupyterLab.`, as it is now more than the server extension. (from the topic of the upstream github repo)